### PR TITLE
Fix the example of using --rsyncdir option

### PR DESCRIPTION
**Pull request by auziel**

### CODE REVIEW

File: docs/remote.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ tests that you can successfully run locally. And you
 have a ssh-reachable machine ``myhost``.  Then
 you can ad-hoc distribute your tests by typing::
 
-    pytest -d --tx ssh=myhostpopen --rsyncdir mypkg mypkg
```
Comment (auziel) [inline docs/remote.rst:13]: the --rsyncdir accept the directory, the last "mypkg" is not needed and misleading.

**Review by nicoddemus — APPROVED** [review]

**Review by RonnyPfannschmidt — CHANGES_REQUESTED** [review]
Ill have to recheck, but I'm pretty sure both are needed, one for the rsync, one for the intended folder to run test's in

**Comment by RonnyPfannschmidt** [comment]
I reread the complete documentation, the name is passed intentionally

**Comment by auziel** [comment]
I will test that again. But when I used `--rsyncdir mypkg mypkg` the test didn't run as expected, everything is working well with `--rsyncdir mypkg`
when I run:
`pytest mypkg/tests/unit_tests -v -s -d --tx socket=<ip>:<port> --rsyncdir mypkg mypkg`
pytest set rootdir to "mypkg" and therefore all the tests under "mypkg/tests" are executed. 

But when I run:
`pytest mypkg/tests/unit_tests -v -s -d --tx socket=<ip>:<port> --rsyncdir mypkg`
Only the tests under "mypkg/tests/unit_tests" are executed.

This is why I thought the doc is misleading.

**Comment by RonnyPfannschmidt** [comment]
Your mistake was leaving mypkg in in your own call

We should perhaps expand the examples to call a specific test instead of all of mypkg

**Comment by auziel** [comment]
@RonnyPfannschmidt thanks for your time.

I'm sorry, but I don't understand your last comment.
What arguments I should use to execute tests under a specific folder?
Can you please give an example for the correct way to execute the tests?

File: docs/remote.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ tests that you can successfully run locally. And you
...
+    pytest -d --tx ssh=myhostpopen --rsyncdir mypkg
```
Comment (RonnyPfannschmidt) [inline docs/remote.rst:13]:
```suggestion
    pytest -d  --rsyncdir mypkg --tx ssh=myhostpopen mypkg/tests/unit/test_sometihng.py
```

might drive it home better, the rsyncdir is indeed a rather confusing option, personally i never used it

Comment (auziel) [reply docs/remote.rst:13]: Great, this makes more sense.

**Comment by nicoddemus** [comment]
Is this OK now @RonnyPfannschmidt ?

**Review by RonnyPfannschmidt — APPROVED** [review]

**Comment by nicoddemus** [comment]
Thanks folks!

**Pull request merged**